packages/p2p/src/components/routes/binary-routes.tsx

```diff
@@ -1,21 +1,24 @@
 import React from 'react';
 import { Switch } from 'react-router-dom';
+import { TRoute } from '@deriv/shared';
 import { Localize } from '@deriv/translations';
 import getRoutesConfig from 'Constants/routes-config';
-import RouteWithSubRoutes from './route-with-sub-routes';
+import { TBinaryRoutes } from 'Types/routes.types';
```

Comment:
can we have index file under types so it's easier to import?

Reply:
sure we could, but wouldn't it be better to have specific routes to specific type files? I think the same is being applied for constants and utils

Reply:
It's easier to read if we change it to `import type { TBinaryRoutes } from Types`. It's better to avoid the redundancy in `Types/routes.types` when importing.